src/components/annotations/draw.js

```diff
@@ -106,12 +116,33 @@ function drawOne(gd, index) {
             });
         });
 
-    // another group for text+background so that they can rotate together
-    var annTextGroup = annGroup.append('g')
-        .classed('annotation-text-g', true)
-        .attr('data-index', String(index));
-
-    var annTextGroupInner = annTextGroup.append('g');
+    if(options.hovertext) {
+        annTextGroupInner
+        .on('mouseover', function() {
+            var hoverOptions = options.hoverlabel;
+            var hoverFont = hoverOptions.font;
+            var bBox = this.getBoundingClientRect();
+            var bBoxRef = gd.getBoundingClientRect();
+
+            Fx.loneHover({
```

Comment:
![peek 2017-04-10 12-07](https://cloud.githubusercontent.com/assets/6675409/24871026/4ba4ee2a-1de6-11e7-8052-c996fed57f58.gif)

so this means the annotation hover takes precedence over trace hover. I don't think this a big deal. But perhaps we make annotation hover pass by `Fx.hover` so that they don't _conflict_ sort of speak.